plugins/helphub-post-types/assets/css/admin.css

```diff
@@ -1,9 +1,612 @@
 .post-type.edit-php table.wp-list-table .column-image {
-	width: 120px;
-	text-align: center;
+  width: 120px;
+  text-align: center; 
 }
 
 .post-type.edit-php table.wp-list-table .column-image img {
 	width: 60px;
-	height: auto;
+	height: auto; 
 }
+
+
```

Comment:
Cool, I've updated the CSS to meet WP's coding standards, will send a PR once this PR is merged so I can squash the comit